cleanlab/regression/rank.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+from cleanlab.outlier import OutOfDistribution
+from sklearn.neighbors import NearestNeighbors
+from cleanlab.internal.regression_utils import assert_valid_inputs
+from typing import Dict, Callable
+from numpy.typing import ArrayLike
+
+""" Generates label quality scores for every sample in regression dataset """
+
+EPS = 1e-30
```

Comment:
add comment to explain in a couple words

Reply:
added here: [f4571b9](https://github.com/cleanlab/cleanlab/pull/572/commits/f4571b9a14b97c5c589bcd9371faf14adc004cd6)